docs/openlaw-object/README.md

```diff
@@ -0,0 +1,2141 @@
+| `jsTemplates` | `Object` | **Required.** An object containing the compiled templates that are linked to a [deal](/markup-language/#deals) template. The object will be empty for non-deal templates. |
+| `jsParams` | `Object` | **Required.** The parameters of the template to be executed. |
+
+Example
+
+```
+const compiledTemplate = Openlaw.compileTemplate("This Advisor Agreement is entered into between [[Company Name]]...");
+const params = {
+  Company Name: "ABC, Inc.",
+  Effective Date: "1537254000000",
+  Number of Shares: "1000",
+  Years Vesting: "4",
+  Unit of Vesting: "250",
+  ...
+};
+Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+```
+
+**Response**
+
+Returns an object containing a [`TemplateExecutionResult` object](#templateexecutionresult) (and key/value pairs indicating that no error has occurred and that no templates are missing) if execution is successful.
+
+Example
+
+```js
+{
+  executionResult: TemplateExecutionResult,
+  isError: false,
+  missingTemplate: false,
+  errorMessage: ""
+}
+```
+
+### executeForReview
+
+Execute (load and run) a compiled template for review which includes data on contract signatories.
+
+```scala
+executeForReview(
+  compiledTemplate: CompiledTemplate,
+  names: js.Dictionary[String],
+  jsTemplates: js.Dictionary[CompiledTemplate],
+  jsParams: js.Dictionary[Any]
+): js.Dictionary[Any]
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `compiledTemplate` | [`CompiledTemplate`](#compiledtemplate) | **Required.** The nested object returned from the [`compileTemplate` method](#compiletemplate). |
+| `names` | `Object` | **Required.** The ID and name of each signatory as a key/value pair. |
+| `jsTemplates` | `Object` | **Required.** An object containing the compiled templates that are linked to a [deal](/markup-language/#deals) template. The object will be empty for non-deal templates. |
+| `jsParams` | `Object` | **Required.** The parameters of the template to be executed for review. |
+
+Example
+
+```
+const compiledTemplate = Openlaw.compileTemplate("This Advisor Agreement is entered into between [[Company Name]]...");
+const signatures = {
+  8f26427b-0853-469b-a4f1-132190b7373e: "openlawuser+1",
+  38e0eb6b-0d52-4fd8-a77d-19686fd3843a: "openlawuser+2"
+};
+const params = {
+  Company Name: "ABC, Inc.",
+  Effective Date: "1537254000000",
+  Number of Shares: "1000",
+  Years Vesting: "4",
+  Unit of Vesting: "250",
+  ...
+};
+Openlaw.executeForReview(compiledTemplate.compiledTemplate, signatures, {}, params);
+```
+
+**Response**
+
+Returns an object containing a [`TemplateExecutionResult` object](#templateexecutionresult) (and key/value pairs indicating that no error has occurred and that no templates are missing) if execution is successful. The [`TemplateExecutionResult` object](#templateexecutionresult) contains a nested object with the ID and name of each signatory as a key/value pair.
```

Comment:
I am not sure about the ordering here. It might be clearer to explain what the `TemplateExecutionResult` object is before talking about the key/value pairs which indicate errors have occurred.

Reply:
Changed the order around to make this clearer.